components/Utils/Footer/index.tsx

```diff
@@ -121,6 +125,15 @@ const Footer = () => {
                   </Button>
                 </Link>
               </li>
+              <li>
+                <Button
+                color='link'
+                className='text-secondary p-0 rounded-0'
+                onClick={openMembership}
+                >
+                  Join
```

Comment:
probably change this to "Join Us" to be consistent with the one above it "Contact Us"??

Reply:
Agree, that would look better